src/Router.js

```diff
@@ -1,11 +1,17 @@
-import { Routes, Route } from 'react-router-dom'
-import Home from './pages/Home/Home'
+import Home from "./pages/Home/Home";
+import { Route, Routes } from 'react-router-dom'
+import Cart from './pages/Cart/Cart'
+import Product from './pages/Product/Product'
+import Wishlist from './pages/Wishlist/Wishlist'
+import Login from './pages/Login/Login'
 export default function Routers(){
     return(
         <Routes>
-            <Route path="/home">
-                <Home />
-            </Route>
-        </Routes>
+            <Route path="home" element={<Home />}/>
```

Comment:
You can give a fallback route using * and render a page not found component.
This will be the default route if none of the above routes match.